docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -1,466 +1,1741 @@
-.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-.. SPDX-License-Identifier: Apache-2.0
-
-Participant Query Store User Guide
-##################################
+PQS Technical User Documentation
+================================
 
 Introduction
-************
+------------
+
+PQS (Participant Query Store) is a component that extends the reach of
+the ledger - by making it available to SQL. This enables users to use
+SQL to query the state of the ledger, as well as the history of how that
+state came to be.
+
+Audience
+~~~~~~~~
+
+This documention provides guidance on what PQS achieves, how to use it,
+and it's operation and limitations. The following types of users are
+likely to find PQS useful:
+
+-  Application developers: to query the ledger to display information to
+   users, integrate or carry out automated business processes, and
+   debugging.
+-  Business analysts: to access and analyze ledger data with ad-hoc
+   investigations to understand operational usage.
+-  Support analysts: to diagnose any problems on a live system, when
+   sophisticated inspection of the ledger is required.
+-  Application operators: to take a snapshot of the ledger, either with
+   or without history in order to analyse and investigate.
+-  Report analysts: to provide access to ledger data information for
+   analysis and aggregation with standard reporting tools, on a
+   historical or real-time basis.
+
+Purpose
+~~~~~~~
+
+PQS can be described as a kind of
+`bridge <https://en.wikipedia.org/wiki/Bridge_pattern>`__ connecting the
+ledger with a datastore. Providing read access to the data via a
+traditional SQL database decouples the reader from a specific
+Participant and enables users to leverage their existing tools and
+knowledge.
+
+The Participant's primary role is to submit and propogate transactions
+in the network, which requires interaction to submit commands. PQS
+sources the transaction ledger from the Participant on a streaming
+basis, maintains history and state based on the events, and makes the
+data available for users to query. Therefore, it alleviates the need for
+the Participant to be involved in read-loads and can be scaled
+independently.
+
+Operational Architecture
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+PQS is a standalone application that is configured via command line
```

Comment:
I may be too sensitive but can we change "application" to "process" because "application implies individuality from the PN which is not what we. are trying to suggest?

Reply:
Yeah application is wrong - I've changed to service as this is more abstract.